### PR TITLE
update babel 6 -> 7

### PR DESCRIPTION
**Pull request by shokai**
- updated babel 6.26.0 -> 7.4.3
- use `@babel/preset-env` instead of `babel-preset-es2015`
- updated mocha to fix github's vulnerability alert
  - Because it is a lodash vulnerability that mocha requires

## how to check

- CI pass
  - In #8, I changed the compiled code to the test target

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "pretest": "npm run build",
     "test": "run-s test:*",
     "test:standard": "standard",
-    "test:mocha": "mocha test/*.js -r babel-register -r babel-polyfill --timeout 20000 --exit",
```
Comment (shokai) [inline package.json]: There is no asynchronous processing in this library, so the timeout option should be its default value.

File: .babelrc [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": "40",
+          "safari": "8",
+          "firefox": "40"
+        }
+      }
+    ]
```
Comment (shokai) [inline .babelrc]: I think this is enough as a target of the library for client js.

**Comment by shokai** [comment]
できた

**Comment by daiiz** [comment]
LGTM

**Pull request merged**